src/components/Label/Label.tsx

```diff
@@ -151,16 +157,15 @@ export const Label = React.forwardRef<HTMLDivElement, LabelProps>(function Label
         event.stopPropagation();
     };
 
-    const labelWithCopy =
-        typeCopy && copyText && !disabled ? (
+    if (hasCopy && copyText) {
```

Comment:
```suggestion
    if (hasCopy) {
```

Reply:
I check copyText value because typescript highlights text prop of the CopyToClipboard below

Reply:
I thought about this double check but I'm think it's ok